### ISSUE TITLE
Add support for Webpack 2

### ISSUE THREAD

**Issue by datakurre** [collective/plonetheme.webpacktemplate#8]
FYI. I'm working on Webpack 2 support.

The current status is that I'm aware of all the related changes between Webpack 1.x and 2.x and working Webpack 2.x build locally, but still need to clean up things for release.

**Comment by ebrehault** [comment]
@datakurre excellent!

By the way, I have just moved this repository to collective, it makes more sense.

**Comment by datakurre** [comment]
Thanks! And if you have permissions, please, also enable Travis-CI at collective :)

I'll move the Webpack plugin later, once version for Webpack 2 is ready. Then I branch all Webpack 1.x stuff as 1.x and masters will be for 2.x. /cc @sneridagh

**Comment by ebrehault** [comment]
Travis is on :)

**Comment by datakurre** [comment]
Merged into master.
